driver/ppm_fillers.c

```diff
@@ -1545,6 +1545,17 @@ static int f_proc_startupdate(struct event_filler_arguments *args)
 		res = val_to_ring(args, tty_nr, 0, false, 0);
 		if (unlikely(res != PPM_SUCCESS))
 			return res;
+
+		/*
+		 * pgrp
+		 */
+#if LINUX_VERSION_CODE > KERNEL_VERSION(2, 6, 24)
```

Comment:
I think this should be `>=`. According to https://elixir.free-electrons.com/linux/v2.6.24/ident/process_group there's no `process_group` function in 2.6.24